fio.h

```diff
@@ -629,7 +630,7 @@ static inline bool multi_range_trim(struct thread_data *td, struct io_u *io_u)
 
 static inline bool should_fsync(struct thread_data *td)
 {
-	if (td->last_was_sync)
+	if (td->last_ddir_issued == DDIR_SYNC)
```

Comment:
With ` last_was_sync` we had:

```
td->last_was_sync = ddir_sync(io_u->ddir);
```

So this should use `ddir_sync()` instead of checking for equality with `DDIR_SYNC` because we actually have three different sync DDIRs.

Reply:
Agreed! will fix it. Thanks.